example/presto_standalone/main.tf

```diff
@@ -0,0 +1,151 @@
+locals {
+  nomad_namespace   = "default"
+  nomad_datacenters = ["dc1"]
+
+  presto = {
+    shared_secret_provider   = "user"
+    vault_kv_policy_name     = "kv-secret"
+    vault_kv_path            = "secret/data/presto"
+    vault_kv_secret_key_name = "cluster_shared_secret"
+    service_name             = "presto"
+  }
+
+  hivemetastore = {
+    service_name = module.hive.service_name
+    port         = 9083 # todo: add output var
+  }
+
+  minio = {
+    service_name = module.minio.minio_service_name
+    port         = 9000 # todo: add output var
+    access_key   = module.minio.minio_access_key
+    secret_key   = module.minio.minio_secret_key
+  }
+}
+
+module "presto" {
+  depends_on = [
+    module.minio,
+    module.hive
+  ]
+
+  source = "../.."
+
+  nomad_job_name         = local.presto.service_name
+  nomad_datacenters      = local.nomad_datacenters
+  nomad_namespace        = local.nomad_namespace
+  shared_secret_provider = local.presto.shared_secret_provider
+
+  shared_secret_vault = {
+    vault_kv_policy_name     = local.presto.vault_kv_policy_name
+    vault_kv_path            = local.presto.vault_kv_path
+    vault_kv_secret_key_name = local.presto.vault_kv_secret_key_name
+  }
+
+  service_name          = local.presto.service_name
+  mode                  = "standalone"
+  workers               = 1
+  consul_http_addr      = "http://10.0.3.10:8500"
+  debug                 = true
+  use_canary            = false
```

Comment:
Should be = true?

Reply:
Honestly I just copy pasted this part. I mean, yeah, we can enable it by default

Reply:
Set to true in both examples